src/styles/blocks/_calendar.scss

```diff
@@ -1,14 +1,62 @@
-@import "variables";
+@import "./../variables";
+@import "./../mixins";
+
+.container {
+  display: flex;
```

Comment:
your whole element sticks out - has a scroll bar - due to default body margins - reset it
then you will have a true 100vh without sticking out
![Snapshot 2022-04-01 15 36 15](https://user-images.githubusercontent.com/39308717/161265111-441c82bd-df01-4ab2-9a24-03adf3dae898.png)

Reply:
fixed